recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "pdbfixer" %}
-{% set version = "1.8.1" %}
+{% set version = "1.9" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/openmm/pdbfixer/archive/v{{ version }}.tar.gz
-  sha256: d50551abfe9dbaefc066f4d9d400cdebe57f1fefd9de9d01e12beb87efd99595
+  git_url: https://github.com/openmm/{{ name }}.git
+  git_rev: "6bf10e1"
```

Comment:
Can we use [the tag instead](https://github.com/openmm/pdbfixer/releases/tag/1.9)?

Reply:
```suggestion
  git_rev: "{{ version }}"
```

Reply:
I find it's best to always specify revisions by hash.  When we try to build conda packages, something often goes wrong that ends up requiring a change to the code, not just the recipe.  So then we have to fix it and update the revision we build against.  Usually I don't create the github release until the conda packages are successfully built.  I did this time just because I wasn't thinking about it.  Sorry about that!